tests/doxy.config.in

```diff
@@ -0,0 +1,2362 @@
+# Doxyfile 1.8.9.1
```

Comment:
I don't understand why there is documentation for the tests.

Reply:
It's so I can follow which aspects of the specs we are validating and where they are tested.  The easiest way was to include something which could be tracked (the @cellml2_# aliases) in the code but seen and sorted from outside.  Once we're sure that everything is validated and covered we the aliases can be removed (or not manually added in the first place) as users probably don't really care about that kind of stuff ...

Reply:
Removed now.

Reply:
Removed now :)